readme.md

```diff
@@ -54,7 +68,7 @@ console.log(results);
 
 Your Answer:
 ```text
-
+This looks ok to me. What is wrong? Maybe needs a promise.
 ```
```

Comment:
`.find` is an asynchronous method, so as the code stands `results` is undefined. We tend to use a callback to fix this (although the latest version of Mongoose lets you use promises). So...

```js
var results = AuthorModel.find({name: "Bob"}, function(results){
  console.log(results);
});
```